examples/scripts/splsrtm_2D.jl

```diff
@@ -66,12 +66,25 @@ end
 bregopt = bregman_options(maxIter=niter, verbose=2, quantile=.9, alpha=.1, antichatter=false, spg=true)
 solb = bregman(obj, zeros(Float32, prod(model0.n)), bregopt, C);
 
-
 # Save final velocity model, function value and history
 h5open("lsrtm_marmousi_breg_result.h5", "w") do file
     write(file, "x", reshape(solb.x, model0.n), "z", reshape(solb.z, model0.n), "fval", Float32.(solb.ϕ_trace))
 end
 
 # Plot final image
 figure(); imshow(copy(adjoint(reshape(solb.x, model0.n))), extent = (0, 7.99, 3.19, 0), cmap = "gray", vmin = -3e-2, vmax = 3e-2)
-title("LS-RTM with SGD"); xlabel("Lateral position [km]"); ylabel("Depth [km]")
+title("SPLS-RTM with SGD"); xlabel("Lateral position [km]"); ylabel("Depth [km]")
+
+#' LSQR
```

Comment:
Can you please try to make a proper example. It's supposed to help new users so hiding this at the bottom of a file about sparsity isn't very friendly.